packages/aws-rfdk/lib/lambdas/nodejs/x509-certificate/test/acm-handlers.test.ts

```diff
@@ -0,0 +1,459 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as AWS from 'aws-sdk';
+import * as AWSMock from 'aws-sdk-mock';
+import * as sinon from 'sinon';
+import { BackoffGenerator } from '../../lib/backoff-generator';
+import { CompositeStringIndexTable } from '../../lib/dynamodb';
+import { Certificate } from '../../lib/x509-certs';
+import { AcmCertificateImporter } from '../acm-handlers';
+import { IAcmImportCertProps } from '../types';
+
+describe('AcmCertificateImporter', () => {
+  const physicalId = 'physicalId';
+  const certArn = 'certArn';
+  const oldEnv = process.env;
+  let consoleWarnSpy: jest.SpyInstance;
+
+  beforeAll(() => {
+    jest.spyOn(global.console, 'log').mockImplementation(() => {});
+    jest.spyOn(global.console, 'error').mockImplementation(() => {});
+    consoleWarnSpy = jest.spyOn(global.console, 'warn').mockImplementation(() => {});
+  });
+
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  beforeEach(() => {
+    process.env.DATABASE = 'database';
+    AWSMock.setSDKInstance(AWS);
+  });
+
+  afterEach(() => {
+    process.env = oldEnv;
+    sinon.restore();
+    AWSMock.restore();
+  });
+
+  describe('doCreate', () => {
+    const doCreateProps: IAcmImportCertProps = {
+      Tags: [],
+      X509CertificatePem: {
+        Cert: 'cert',
+        CertChain: 'certChain',
+        Key: 'key',
+        Passphrase: 'passphrase',
+      },
+    };
+
+    beforeEach(() => {
+      sinon.stub(Certificate, 'decryptKey').returns(new Promise((res, _rej) => res('key')));
```

Comment:
nit: minor simplification

```suggestion
      sinon.stub(Certificate, 'decryptKey').returns(Promise.resolve('key'));
```

Reply:
Neat, thanks!